src/components/admin/article/ArticleController.jsx

```diff
@@ -273,8 +273,8 @@ export default class ArticleController extends FalcorController {
       const article = this.state.data.articles.bySlug[slug];
 
       // If it is a new article it won't have any meta data yet so we use the default
-      const categories = this.state.data.categories.byIndex;
-      categories.none = { name: 'none', slug: 'none' };
+      const categories = _.map(this.state.data.categories.byIndex);
```

Comment:
Oh didn't actually know that _.map has the default callback function as the identity? By this behaviour it seems it does?

Reply:
It does, but just realized there is a better function for this.